base/system/services/rpcserver.c

```diff
@@ -5120,7 +5121,7 @@ RChangeServiceConfig2A(
 
             lpServiceDescriptionW = HeapAlloc(GetProcessHeap(),
                                               HEAP_ZERO_MEMORY,
-                                              dwLength + sizeof(SERVICE_DESCRIPTIONW));
+                                              sizeof(SERVICE_DESCRIPTIONW) + dwLength);
```

Comment:
Unnecessary

Reply:
I will remove it.
NB:
I concur. Yet, I don't understand the value of (keeping) inconsistent code: it just makes things somewhat obscure to me...
See `5129 lpServiceDescriptionW->lpDescription = (LPWSTR)(lpServiceDescriptionW + 1);`.